### PR TITLE
bugfix: Discord notification message

### PR DESCRIPTION
**Pull request by lukman7788**
# Monika Pull Request (PR)  

## What feature/issue does this PR add  
1. Add text `New Recovery event from Monika` or `New Incident event from Monika` on discord message

## How did you implement / how did you fix it  
1.  Add parameter to function
2. Add some text on message body

image

### CODE REVIEW

**Review by dennypradipta — APPROVED** [review]

**Review by nicnocquee — APPROVED** [review]

File: .vscode/launch.json [hunk 1]
```diff
@@ -7,10 +7,15 @@
...
-      "name": "Launch Insomnia",
+      "name": "Launch Symon",
       "skipFiles": ["<node_internals>/**"],
       "program": "${workspaceFolder}\\bin\\run_dev",
-      "args": ["-I", "./insomnia.json"],
+      "args": [
+        "--symonUrl",
+        "http://localhost:3000/",
+        "--symonKey",
+        "dc5b4caa-46f1-435f-9e23-7141b9b1a85a"
```
Comment (gsukanto) [inline .vscode/launch.json:17]: What will happen if other developer use vscode to run Monika?
Do they need to run symon in their localhost with this `symonKey`?

File: src/components/notification/channel/discord.ts [hunk 2]
```diff
@@ -25,10 +25,29 @@
...
+  message: NotificationMessage
+): Promise<any> => {
   try {
+    const notificationType =
+      message.meta.type[0].toUpperCase() + message.meta.type.substring(1)
+    let content
+    switch (message.meta.type) {
+      case 'incident':
+      case 'recovery': {
+        content = `New *\`${notificationType}\`* event from Monika\n\n${message.body}`
```
Comment (gsukanto) [inline src/components/notification/channel/discord.ts:42]: Why do we use double newline here `\n\n` when we will trim it as one newline `\n` in the next line?

**Review by gsukanto — APPROVED** [review]
lgtm

File: .vscode/launch.json (hunk 1 above)
Comment (lukman7788) [reply .vscode/launch.json]: updated @gsukanto

File: src/components/notification/channel/discord.ts (hunk 2 above)
Comment (lukman7788) [reply src/components/notification/channel/discord.ts]: updated @gsukanto

**Pull request merged**